src/net.h

```diff
@@ -650,6 +651,7 @@ class CNode
     const NodeId id;
 
     const uint64_t nKeyedNetGroup;
+    std::atomic_bool fPauseRecv;
```

Comment:
Grr, another artifact of breaking up an end-result into smaller commits. I'll add this where it belongs in the next rebase.